src/DotNetty.Transport/Channels/Groups/DefaultChannelGroup.cs

```diff
@@ -310,18 +310,19 @@ public Task NewCloseFuture(IChannelMatcher matcher)
 
         static object SafeDuplicate(object message)
         {
-            if (message is IByteBuffer)
+            var buffer = message as IByteBuffer;
+            if (buffer != null)
             {
-                return ((IByteBuffer)message).Duplicate().Retain();
+                return buffer.Duplicate().Retain();
```

Comment:
this change specifically does nothing, it's minor cleanup. Actual change to fix out of range exceptions are in `EmptyByteBuffer`